integrations/ipfy_test.go

```diff
@@ -0,0 +1,71 @@
+// +build integration
+
+package integrations_test
+
+import (
+	. "friendly-potato/integrations"
+	"net"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestIPFy_GetExtIP(t *testing.T) {
+
+	t.Logf("setup ipfy")
+	f, err := NewIPFy()
+	if err != nil {
+		t.Logf("NewIPFy() failed: %v",err)
+		t.FailNow()
+	}
+
+	tests := []struct {
+		name    string
+		fields  *IPFy
+		wantIp  net.IP
+		wantErr bool
+	}{
+		{
+			name:    "test get external ip",
+			fields:  f,
+			wantIp:  f.Resolved,
+			wantErr: false,
+		},
+	}
+
+	expectedKnownIP, ok := os.LookupEnv("EXTERNAL_IP")
+	if !ok {
+		t.Log("EXTERNAL_IP env not set skipping")
+	} else {
+		extTest := struct {
+			name    string
+			fields  *IPFy
+			wantIp  net.IP
+			wantErr bool
+		}{
+			name:    "test known address",
+			fields:  f,
+			wantIp:  net.ParseIP(expectedKnownIP),
+			wantErr: false,
+		}
+		tests = append(tests, extTest)
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			i,err := NewIPFy()
+			if err != nil {
+				t.Logf("InitIPFy() failed: %v",err)
+				t.FailNow()
+			}
+			gotIp, err := i.GetExtIP()
```

Comment:
Codacy found an issue: [var gotIp should be gotIP](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)